packages/webview_flutter_lwe/tizen/src/webview.h

```diff
@@ -66,6 +68,7 @@ class WebView : public PlatformView {
   void InitWebView();
 
   LWE::WebContainer* webview_instance_ = nullptr;
+  MessageDispatcher* dispatcher = nullptr;
```

Comment:
This object goes with the lifecycle of the WebView. I think it would be  more efficient to declare it as a unique_ptr?